### ISSUE TITLE
Update homebrew formulae: 1) enable xlsx format, 2) truncate not available in help docs, so something is broken

### ISSUE THREAD

**Issue by reportaman** [andmarti1424/sc-im#491]
Please update the homebrew formulae for sc-im to https://formulae.brew.sh/formula/sc-im#default to enable xlsx support.

As a side note, it also seems like an older version as help file does not have information on configuration flag ``truncate``... Surprisingly, both homebrew and my build version say 0.7.0, so something is broken in versioning, or homebrew somehow picks up stale source code.

Homebrew's version:
<img width="895" alt="Screen Shot 2021-02-23 at 2 23 51 PM" src="https://user-images.githubusercontent.com/71736629/108917578-ce5b1100-75e4-11eb-867b-c8d410d302db.png">

My local build:

<img width="955" alt="Screen Shot 2021-02-23 at 2 39 43 PM" src="https://user-images.githubusercontent.com/71736629/108917700-02363680-75e5-11eb-9a00-a88dca325e8f.png">

**Comment by andmarti1424** [comment]
I believe someone already did it. Now its at v0.8

**Comment by reportaman** [comment]
@andmarti1424 Yes, I confirm that. Though ``sc-im --version`` gives misleading value of ``0.7.0``. Also, the brew bottle (i.e. pre-built binary) doesn't have a dependency on xlsread/write libs, so it doesn't come with those features enabled. And for some reason, doing ``brew install sc-im --build-fromw-source`` gives ``Error: SHA mismatch``:

```
>>brew install sc-im --build-from-source
==> Downloading https://github.com/andmarti1424/sc-im/archive/v0.8.0.tar.gz
==> Downloading from https://codeload.github.com/andmarti1424/sc-im/tar.gz/v0.8.0
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: ba65b3936a21ae65b19d99a4cfbb69bb57d8b00880f0781fb620e8857bc498db
  Actual: edc9b9d2dc5ac37c31a46462a1387605983149a1abba00d47ac22f2726ce6d27
    File: /Users/reportaman/Library/Caches/Homebrew/downloads/4c4937f865725bd4f494a1bd4312156bcd3121e9ee243201fd970f1cd15893ab--sc-im-0.8.0.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by andmarti1424** [comment]
Oh I see. It should be updated I believe. Isn't a maintainer for that? Or a mark to set it outdated just like in AUR?

**Comment by andmarti1424** [comment]
@reportaman now it is at v0.8.1 I believe. Have you tried it?

**Comment by reportaman** [comment]
Yes, now it says 0.8.2-dev

```
>>which sc-im
/opt/homebrew/bin/sc-im
>>sc-im --version
SC-IM - version 0.8.2-dev
```

**Comment by andmarti1424** [comment]
I believe it shouldn't. V0. 8.2 is not a release. 
-------- Original message --------From: reportaman ***@***.***> Date: 4/16/21  20:11  (GMT-03:00) To: andmarti1424/sc-im ***@***.***> Cc: Andrés Martinelli ***@***.***>, Mention ***@***.***> Subject: Re: [andmarti1424/sc-im] Update homebrew formulae: 1) enable xlsx format, 2) truncate not available in help docs, so something is broken (#491) 
Yes, now it says 0.8.2-dev
>>which sc-im
/opt/homebrew/bin/sc-im
>>sc-im --version
SC-IM - version 0.8.2-dev
